### PR TITLE
App.js from class to function

### PR DESCRIPTION
**Pull request by echeveria**

### CODE REVIEW

File: components/App.js [hunk 1]
```diff
@@ -11,223 +11,196 @@ import graphQLFetcher from '../utils/graphQLFetcher';
...
+    };
 
-            <GraphiQL.Menu label="Environment" title="Environment">
-              <GraphiQL.MenuItem label="Prod" title="Prod" onSelect={() => this.handleEnvironmentChange('prod')} />
-              <GraphiQL.MenuItem label="Staging" title="Staging" onSelect={() => this.handleEnvironmentChange('staging')} />
-              <GraphiQL.MenuItem label="Dev" title="Dev" onSelect={() => this.handleEnvironmentChange('dev')} />
-            </GraphiQL.Menu>
+    const onEditQuery = (query) => {
+        setParameters({ ...parameters, query});
+        this.updateURL();
```
Comment (draperunner) [inline components/App.js]: There's no `this.` anymore. You should simply call `updateURL()`

**Comment by echeveria** [comment]
For thus code I use github browser editor. May be tomorrow I will use linter on my computer IDE.

File: components/App.js [hunk 2]
```diff
@@ -11,223 +11,196 @@ import graphQLFetcher from '../utils/graphQLFetcher';
...
-      </GraphiQL.Menu>
-    )
-  }
-
-  render() {
-    const { parameters, isConfigLoaded } = this.state;
-
-    if (!isConfigLoaded) {
-      return <div>Loading ...</div>;
+        localStorage.removeItem('graphiql:query');
```
[the remark this answers is not in the document]
Comment (echeveria) [reply components/App.js]: >"npm run lint"  are you sure?

[the remark this answers is not in the document]
Comment (draperunner) [reply components/App.js]: Yes. You will have to update your fork with the master of this original repo to get the latest changes. 

If you wonder how to do that, I can recommend my own tool [`git-pretty`](https://github.com/draperunner/git-pretty) which has a tutorial for syncing a fork ;)

**Comment by draperunner** [comment]
Looks like some changes to the GeocoderModal files appeared after the merge. Please revert them.

Also, you can run `npm run lint -- --fix` to automatically resolve lint errors. There should be no semicolons.

**Comment by echeveria** [comment]
--fix is bad idea for me but... OK :)

File: components/App.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (echeveria) [reply components/App.js]: Hi, in my hacktober dashboard all issue is "Pending"?

[the remark this answers is not in the document]
Comment (draperunner) [reply components/App.js]: Hmm, this PR is approved and merged, so I don't know what's wrong. I guess you should contact the Hacktoberfest organizers and ask them.

**Pull request merged**